dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -303,7 +304,7 @@ export const CommentLayout = ({ article, NAV, format }: Props) => {
 
 	const contributionsServiceUrl = getContributionsServiceUrl(article);
 
-	const renderAds = !article.isAdFreeUser && !article.shouldHideAds;
+	const renderAds = hasAds(article);
```

Comment:
Minor suggestion which you can ignore but could the function name and conditional reflect each other a little more e.g.:

```suggestion
	const renderAds = canRenderAds(article);
```

Reply:
yeah i went through several iterations of this - happy with whatever people prefer